### PR TITLE
Rollup gopherage typescript bundle with make rule

### PR DESCRIPTION
**Pull request by chaodaiG**
This is an example from reverse engineering rollup_bundle bazel target.

Gopherage typescript bundle is already source code controlled, verify-gopherage script is meant to ensure that the bundled js file is correct, however https://github.com/kubernetes/test-infra/blob/26e57a2169f9f24d8980bdf63c6bb268b3522ea1/hack/verify-gopherage.sh#L36 doesn't really catch error when the generated file doesn't exist, which doesn't really work.

Part of https://github.com/kubernetes/test-infra/issues/23897
/cc @BenTheElder

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "@rollup/plugin-commonjs": "14.0.0",
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-node-resolve": "8.4.0",
+    "rollup-plugin-terser": "7.0.2",
```
Comment (chaodaiG) [inline package.json:21]: Failed to discover the version of terser used by bazel, so picked the latest version of this plugin. Will take a further look trying to align as much as possible

File: gopherage/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { terser } from "rollup-plugin-terser";
+
+export default {
+  context: "window",
+  input: './gopherage/cmd/html/static/_output/browser.js',
+  output: [
+    {
```
Comment (chaodaiG) [inline gopherage/rollup.config.js:9]: Double output pretty much from https://github.com/kubernetes/test-infra/blob/26e57a2169f9f24d8980bdf63c6bb268b3522ea1/def/ts.bzl#L35

File: package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "@rollup/plugin-commonjs": "14.0.0",
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-node-resolve": "8.4.0",
+    "rollup-plugin-terser": "5.0.0",
```
Comment (chaodaiG) [inline package.json:21]: expect `terser 4.4.0` as defined below. Did a binary search https://openbase.com/js/rollup-plugin-terser/versions:

5.1.0 -> terser 4.1.0
5.1.2 -> terser 4.1.0
5.1.3 -> terser 4.1.0
5.2.0 -> terser 4.6.2

The closest one is 5.1.3

File: package.json [hunk 4]
```diff
@@ -18,6 +18,7 @@
     "@rollup/plugin-commonjs": "14.0.0",
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-node-resolve": "8.4.0",
+    "rollup-plugin-terser": "5.1.3",
```
Comment (chaodaiG) [inline package.json:21]: expect terser 4.4.0 as defined below. Did a binary search https://openbase.com/js/rollup-plugin-terser/versions:

5.1.0 -> terser 4.1.0
5.1.2 -> terser 4.1.0
5.1.3 -> terser 4.1.0
5.2.0 -> terser 4.6.2

The closest one is 5.1.3

File: gopherage/Makefile [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include ../Makefile.base.mk
+# Makefile.base.mk requires REPO_ROOT to be set
+REPO_ROOT:=${CURDIR}/..
+
+rollup-js:
```
Comment (BenTheElder) [inline gopherage/Makefile:19]: this makefile is a good target for having non-phony "output file depends on rules & input files" (which also allows make to skip running when mtime of output is >= inputs), but we can refactor later.

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve

**Pull request merged**